app/picture-answer/picture-answer.html

```diff
@@ -0,0 +1,5 @@
+<div ng-class="selected: false">
+  <button src="pic.jpg">
```

Comment:
Buttons don't have `src`'s. Ditch that attribute, and the image file, too, please.